src/DatabaseSetup.php

```diff
@@ -115,20 +115,32 @@ public function disableRemoteAccess()
      */
     public function addRemoteAccessUser()
     {
-        $input = $this->climate->lightBlue()->input("What is the IP address of the remote server that will be accessing the database?");
+        $input = $this->climate->lightBlue()->input("Enter 'd' for Sonar defaults or IP address of the remote server that will be accessing the database.");
+        $ipAddresses = [];
         $ipAddress = null;
         while ($ipAddress == null || filter_var($ipAddress, FILTER_VALIDATE_IP) === false)
         {
             $ipAddress = $input->prompt();
             if ($ipAddress == null)
             {
-                $this->climate->shout("You must input an IP address.");
+                $this->climate->shout("You must input an IP address or 'd' for Sonar defaults.");
+            }
+            elseif ($ipAddress === 'd')
+            {
+                $this->climate->lightMagenta("Using egress IPs of Sonar..");
+                $ipAddresses = ['20.221.112.37', '20.221.114.13', '52.158.209.86'];
```

Comment:
Just more of a question that we are OK exposing these in a public repo.

Reply:
These are public and meant to be consistent so that customers can set appropriate firewall rules: https://docs.sonar.expert/networking/sonar-ip-addressing